pkg/virt-handler/vm.go

```diff
@@ -1134,9 +1134,19 @@ func (d *VirtualMachineController) updateIsoSizeStatus(vmi *v1.VirtualMachineIns
 		return
 	}
 
+	volumes := map[string]*v1.Volume{}
```

Comment:
Before, it was assumed that there is a Disk <-> Volume relationship, where all volumes have a disk associated with them. But, the right thing to assume is just Disk -> Volume. IMHO I think it is much clearer (and less error-prone) to go through the Disks, since these are the only ones that has an .iso, and extract the information from the associated volume. Than to assume that volumes have a Disk except those that have a Filesystem.